mk/sm.spec.in

```diff
@@ -43,6 +43,9 @@ sed -i 's/archive = .*/archive = 0/' /etc/lvm/master/lvm.conf || exit $?
 sed -i 's/filter \= \[ \"a\/\.\*\/\" \]/filter = \[ \"r\|\/dev\/xvd\.\|\"\, \"r\|\/dev\/VG\_Xen\.\*\/\*\|\"\]/g' /etc/lvm/master/lvm.conf || exit $?
 cp /etc/lvm/master/lvm.conf /etc/lvm/lvm.conf || exit $?
 sed -i 's/metadata_read_only =.*/metadata_read_only = 1/' /etc/lvm/lvm.conf || exit $?
+sed -i 's/use_lvmetad = 1/use_lvmetad = 0/' /etc/lvm/lvm.conf
+sed -i 's/use_lvmetad = 1/use_lvmetad = 0/' /etc/lvm/master/lvm.conf
+ln -sf /dev/null /etc/udev/rules.d/69-dm-lvm-metad.rules
```

Comment:
This needs to go in the Makefile. Check the code for 62-multipath.
It does not build otherwise